docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -483,6 +484,25 @@ public void startContainer(String id, CubeContainer containerConfiguration) {
         }
     }
 
+    public Statistics statsContainer(String id) {
+         this.readWriteLock.readLock().lock();
+
+        try {
+            StatsCmd statsCmd = this.dockerClient.statsCmd(id);
+            StatsLogsResultCallback statslogs = new StatsLogsResultCallback();
+            try {
+                statsCmd.exec(statslogs).awaitCompletion();
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```

Comment:
We should log and throw exception here. Otherwise we can end up with inconsistent state.